src/widgets/FileWidget/index.js

```diff
@@ -43,56 +67,191 @@ const FileWidget = (props) => {
     placeholder,
   } = props;
 
-  const options = {
-    title: '',
-    takePhotoButtonTitle: 'Take Photo',
-    chooseFromLibraryButtonTitle: 'Add from gallery',
+  const refRBSheet = useRef();
+
+  const isIOS = Boolean(process.env.STORYBOOK_IS_IOS) || Platform.OS === 'ios';
+
+  const requestCameraPermission = async () => {
+    if (!isIOS) {
+      try {
+        const granted = await PermissionsAndroid.request(
+          PermissionsAndroid.PERMISSIONS.CAMERA,
+          {
+            title: 'Camera Permission',
+            message: 'App needs camera permission',
+          },
+        );
+        // If CAMERA Permission is granted
+        return granted === PermissionsAndroid.RESULTS.GRANTED;
+      } catch (err) {
+        console.warn(err);
+        return false;
+      }
+    } else return true;
   };
 
-  const showPicker = () => {
-    ImagePicker.showImagePicker(options, (response) => {
-      if (response.didCancel) {
-        console.log('User cancelled image picker');
-      } else if (response.error) {
-        console.log('ImagePicker Error: ', response.error);
-      } else {
-        // You can also display the image using data:
-        // const source = { uri: 'data:image/jpeg;base64,' + response.data };
-        onChange(response.uri, name);
+  const requestExternalWritePermission = async () => {
+    if (!isIOS) {
+      try {
+        const granted = await PermissionsAndroid.request(
+          PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE,
+          {
+            title: 'External Storage Write Permission',
+            message: 'App needs write permission',
+          },
+        );
+        // If WRITE_EXTERNAL_STORAGE Permission is granted
+        return granted === PermissionsAndroid.RESULTS.GRANTED;
+      } catch (err) {
+        console.warn(err);
+        alert('Write permission err', err);
       }
-    });
+      return false;
+    } return true;
   };
-  const placeholderStyle = theme.input[hasError ? 'error' : 'regular'].placeholder;
 
-  const isIOS = Boolean(process.env.STORYBOOK_IS_IOS) || Platform.OS === 'ios';
+  const onImageReceive = (response) => {
+    console.log('Response = ', response.uri);
+
+    if (response.didCancel) {
+      console.log('User cancelled image picker');
```

Comment:
maybe alert should be here like on the holderapp
![image](https://user-images.githubusercontent.com/26321483/135819755-84c5b768-3378-44e3-9bcd-40c98f8c1cf4.png)